### ISSUE TITLE
Problem with custom installation

### ISSUE THREAD

**Issue by j3gu** [pinellolab/dictys#39]
**Checks before submitting the issue**
* [x] I installed the latest Dictys version successfully
* [x] I downloaded the whole folder for the tutorial
* [] I ran the tutorial cells and notebooks (if more than one) in the correct order and only once
* [ ] The preceeding cells and notebooks finished successfully (i.e. the reported error is the first encountered)

I only have the option of installing Dictys with PIP on the clusters. With current version v1.0.0, I encountered errors on running wellington_footprints.py, which is "TypeError: unsupported operand type(s) for >>: 'builtin_function_or_method' and '_io.TextIOWrapper'. I compared this script with the one download through Conda, and suspected that the version on PIP was not the most updated one. Could you please take a look? Thanks.

Here is a list of the installed packages in the virtual environment. 
 Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
_openmp_mutex             5.1                       1_gnu  
accelerate                0.21.0                   pypi_0    pypi
aiohttp                   3.8.5                    pypi_0    pypi
aiosignal                 1.3.1                    pypi_0    pypi
appdirs                   1.4.4                    pypi_0    pypi
async-timeout             4.0.3                    pypi_0    pypi
attrs                     23.1.0                   pypi_0    pypi
audioread                 3.0.0                    pypi_0    pypi
bitsandbytes              0.41.1                   pypi_0    pypi
blas                      1.0                         mkl  
bottleneck                1.3.5            py39h7deecbd_0  
brotlipy                  0.7.0           py39h27cfd23_1003  
bzip2                     1.0.8                h7b6447c_0  
ca-certificates           2023.08.22           h06a4308_0  
certifi                   2023.7.22        py39h06a4308_0  
cffi                      1.15.1           py39h74dc2b5_0  
charset-normalizer        2.1.1                    pypi_0    pypi
cryptography              41.0.3           py39h130f0dd_0  
cuda-cudart               11.7.99                       0    nvidia
cuda-cupti                11.7.101                      0    nvidia
cuda-libraries            11.7.1                        0    nvidia
cuda-nvrtc                11.7.99                       0    nvidia
cuda-nvtx                 11.7.91                       0    nvidia
cuda-runtime              11.7.1                        0    nvidia
datasets                  2.14.4                   pypi_0    pypi
decorator                 5.1.1                    pypi_0    pypi
dill                      0.3.7                    pypi_0    pypi
ffmpeg                    4.2.2                h20bf706_0  
filelock                  3.8.2                    pypi_0    pypi
freetype                  2.12.1               h4a9f257_0  
frozenlist                1.4.0                    pypi_0    pypi
fsspec                    2023.6.0                 pypi_0    pypi
giflib                    5.2.1                h5eee18b_3  
gmp                       6.2.1                h295c915_3  
gmpy2                     2.1.2            py39heeb90bb_0  
gnutls                    3.6.15               he1e5248_0  
huggingface-hub           0.16.4                   pypi_0    pypi
idna                      3.4              py39h06a4308_0  
intel-openmp              2021.4.0          h06a4308_3561  
jinja2                    3.1.2            py39h06a4308_0  
jpeg                      9e                   h5eee18b_1  
lame                      3.100                h7b6447c_0  
lazy-loader               0.3                      pypi_0    pypi
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.38                 h1181459_1  
lerc                      3.0                  h295c915_0  
libcublas                 11.10.3.66                    0    nvidia
libcufft                  10.7.2.124           h4fbf590_0    nvidia
libcufile                 1.7.1.12                      0    nvidia
libcurand                 10.3.3.129                    0    nvidia
libcusolver               11.4.0.1                      0    nvidia
libcusparse               11.7.4.91                     0    nvidia
libdeflate                1.17                 h5eee18b_0  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 11.2.0               h1234567_1  
libgomp                   11.2.0               h1234567_1  
libidn2                   2.3.4                h5eee18b_0  
libnpp                    11.7.4.75                     0    nvidia
libnvjpeg                 11.8.0.2                      0    nvidia
libopus                   1.3.1                h7b6447c_0  
libpng                    1.6.39               h5eee18b_0  
librosa                   0.10.1                   pypi_0    pypi
libstdcxx-ng              11.2.0               h1234567_1  
libtasn1                  4.19.0               h5eee18b_0  
libtiff                   4.5.1                h6a678d5_0  
libunistring              0.9.10               h27cfd23_0  
libvpx                    1.7.0                h439df22_0  
libwebp                   1.2.4                h11a3e52_1  
libwebp-base              1.2.4                h5eee18b_1  
llvmlite                  0.40.1                   pypi_0    pypi
lz4-c                     1.9.4                h6a678d5_0  
mako                      1.2.4                    pypi_0    pypi
markupsafe                2.1.1            py39h7f8727e_0  
mkl                       2021.4.0           h06a4308_640  
mkl-service               2.4.0            py39h7f8727e_0  
mkl_fft                   1.3.1            py39hd3c417c_0  
mkl_random                1.2.2            py39h51133e4_0  
mpc                       1.1.0                h10f8cd9_1  
mpfr                      4.0.2                hb69a4c5_1  
mpmath                    1.3.0            py39h06a4308_0  
msgpack                   1.0.5                    pypi_0    pypi
multidict                 6.0.4                    pypi_0    pypi
multiprocess              0.70.15                  pypi_0    pypi
ncurses                   6.4                  h6a678d5_0  
nettle                    3.7.3                hbbd107a_1  
networkx                  3.1              py39h06a4308_0  
numba                     0.57.1                   pypi_0    pypi
numexpr                   2.8.4            py39he184ba9_0  
numpy                     1.24.3           py39h14f4228_0  
numpy-base                1.24.3           py39h31eccc5_0  
openh264                  2.1.1                h4ff587b_0  
openssl                   1.1.1w               h7f8727e_0  
packaging                 23.1             py39h06a4308_0  
pandas                    1.5.2            py39h417a72b_0  
pillow                    9.3.0                    pypi_0    pypi
pip                       23.2.1           py39h06a4308_0  
platformdirs              3.10.0                   pypi_0    pypi
pooch                     1.7.0                    pypi_0    pypi
psutil                    5.9.5                    pypi_0    pypi
pyarrow                   12.0.1                   pypi_0    pypi
pycparser                 2.21               pyhd3eb1b0_0  
pycuda                    2022.2.2                 pypi_0    pypi
pyopenssl                 23.2.0           py39h06a4308_0  
pysocks                   1.7.1            py39h06a4308_0  
python                    3.9.12               h12debd9_1  
python-dateutil           2.8.2              pyhd3eb1b0_0  
pytools                   2023.1.1                 pypi_0    pypi
pytorch                   2.0.0           py3.9_cuda11.7_cudnn8.5.0_0    pytorch
pytorch-cuda              11.7                 h778d358_5    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2022.7           py39h06a4308_0  
readline                  8.2                  h5eee18b_0  
regex                     2022.10.31               pypi_0    pypi
requests                  2.28.1                   pypi_0    pypi
safetensors               0.3.2                    pypi_0    pypi
scikit-learn              1.3.0                    pypi_0    pypi
scipy                     1.11.2                   pypi_0    pypi
setuptools                68.0.0           py39h06a4308_0  
six                       1.16.0             pyhd3eb1b0_1  
soundfile                 0.12.1                   pypi_0    pypi
soxr                      0.3.6                    pypi_0    pypi
sqlite                    3.41.2               h5eee18b_0  
sympy                     1.11.1           py39h06a4308_0  
threadpoolctl             3.2.0                    pypi_0    pypi
tk                        8.6.12               h1ccaba5_0  
tokenizers                0.13.3                   pypi_0    pypi
torch                     1.11.0+cu115             pypi_0    pypi
torchaudio                0.11.0+cu115             pypi_0    pypi
torchtriton               2.0.0                      py39    pytorch
torchvision               0.12.0+cu115             pypi_0    pypi
tqdm                      4.64.1                   pypi_0    pypi
transformers              4.31.0                   pypi_0    pypi
typing-extensions         4.4.0                    pypi_0    pypi
typing_extensions         4.7.1            py39h06a4308_0  
tzdata                    2023c                h04d1e81_0  
urllib3                   1.26.13                  pypi_0    pypi
wheel                     0.38.4           py39h06a4308_0  
x264                      1!157.20191217       h7b6447c_0  
xxhash                    3.3.0                    pypi_0    pypi
xz                        5.4.2                h5eee18b_0  
yarl                      1.9.2                    pypi_0    pypi
zlib                      1.2.13               h5eee18b_0  
zstd                      1.5.5                hc292b87_0

**Comment by lingfeiwang** [comment]
Hi j3gu,

From your list of installed packages, you seem to have access to conda installation method. Have you tried that?

Also could you post the full error message?

**Comment by j3gu** [comment]
Yes, I have tried the Conda installation method. I did not have problem running the scripts except for the GPU computation. To run PyTorch, I have to use a pre-installed conda environment, which needs administrator access. 

Here is the full error message:

Traceback (most recent call last):
  File "/home/jinggu/.local/bin/wellington_footprints.py", line 106, in <module>
    print >> wigout, "track type=wiggle_0"
TypeError: unsupported operand type(s) for >>: 'builtin_function_or_method' and '_io.TextIOWrapper'. Did you mean "
print(<message>, file=<output_stream>)"?
